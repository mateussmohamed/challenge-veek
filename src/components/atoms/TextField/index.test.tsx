import React from 'react'

import { TextField } from './index'
import withThemeProvider from '../../theme/withThemeProvider'

describe('[<TextField />]', () => {
  const wrap = props => withThemeProvider(() => <TextField {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ name: 'email', onChange: jest.fn() })

    expect(wrapper).toMatchSnapshot()
  })
})
