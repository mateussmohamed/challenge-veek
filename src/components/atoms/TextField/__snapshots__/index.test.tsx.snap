// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<TextField />] matches to snapshot 1`] = `[Function]`;
